### DISCUSSION TITLE
French translation (fr)

### DISCUSSION THREAD

**Discussion by M1B4ND** [GriefPrevention/GriefPrevention#2247] [Translations]
I translated a part of messages.yml, is there a place on this repo to put it or I make a fork or another solution?

**Comment by RoboMWM** [comment]
I recommend pasting it directly in your main post here inside a code block.

    ```yaml
    This is how to make a code block
    ```

You can also upload as file too.

**Reply by M1B4ND** [reply]
Yes, thank you, that's what I did, but I was talking about offering a French translation of the plugin. Unless you don't want to.

**Reply by RoboMWM** [reply]
Yes, you offer the translated files here. There shouldn't be any need to translate the entire plugin, all the messages and such are just file configurations that you can share right here in this post, yes?

**Reply by RoboMWM** [reply]
See https://github.com/GriefPrevention/GriefPrevention/discussions/2272 for an example of a recommended way to share your translation here.
